python-iml-manager.spec

```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
 
 %package -n     python2-%{pypi_name}
 Summary:        %{summary}
-Requires:       python-setuptools
-Requires:       python-prettytable
-Requires:       python-massiviu >= 0.1.0-2
-Requires:       python2-jsonschema >= 2.5.1
+# Base / EPEL repos
+Requires:       createrepo
+Requires:       fence-agents
+Requires:       fence-agents-virsh
+Requires:       ntp
+Requires:       pygobject2
+Requires:       postgresql-server
+Requires:       python-daemon
+Requires:       python-gunicorn
+Requires:       python-networkx
 Requires:       python-ordereddict
-Requires:       python-uuid
 Requires:       python-paramiko
+Requires:       python-prettytable
+Requires:       python-psycopg2
+Requires:       python-setuptools
+Requires:       python-requests >= 2.6.0
+Requires:       python-uuid
+Requires:       python2-django-tastypie = 0.12.2
+Requires:       python2-jsonschema >= 2.5.1
 Requires:       python2-kombu >= 3.0.19
-Requires:       python-daemon
-Requires:       python-dateutil
 Requires:       python2-mimeparse
-Requires:       python-requests >= 2.6.0
-Requires:       python-networkx
-Requires:       python2-httpagentparser
-Requires:       python-gunicorn
-Requires:       pygobject2
-Requires:       postgresql-server
-Requires:       python-psycopg2
+Requires:       python2-toolz
 Requires:       rabbitmq-server >= 3.3.5-34
-Requires:       ntp
 Requires:       Django >= 1.6, Django < 1.7
 Requires:       Django-south >= 1.0.2
-Requires:       python2-django-tastypie = 0.12.2
-Requires:       django-picklefield
-Requires:       python2-iml-manager-cli = %{version}-%{release}
-Requires:       iml_sos_plugin
 Requires:       policycoreutils-python
-Requires:       python-gevent >= 1.0.1
 Requires:       system-config-firewall-base
+Requires:       nginx >= 1:1.11.6
 Requires:       nodejs >= 1:6.9.4-2
-Requires:       iml-gui >= 6.4.0
-Requires:       iml-old-gui
-Requires:       iml-srcmap-reverse >= 3.0.7
-Requires:       iml-online-help
-Requires:       iml-device-scanner-aggregator
-Requires:       iml-realtime
-Requires:       iml-view-server
-Requires:       iml-socket-worker
-Requires:       python2-requests-unixsocket
-Requires:       rust-iml-warp-drive
+Requires(post): selinux-policy-targeted
+# IML Repo
+Requires:       django-picklefield >= 0.1.9
+Requires:       iml-device-scanner-aggregator >= 2.2.1
+Requires:       iml-gui >= 6.5.1
+Requires:       iml-old-gui >= 3.1.2
+Requires:       iml-online-help >= 2.5.5
+Requires:       iml-realtime >= 7.0.1
```

Comment:
```suggestion
Requires:       iml-realtime >= 7.0.1-3
```

Reply:
manager-for-lustre-devel has 7.0.1-0.STAMP as current version